package.json

```diff
@@ -2,9 +2,10 @@
   "name": "todo-list",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] Nice that you added Webpack to your scripts at the package.json file :+1: , but we need to go further and create at least 2 scripts for your application, one for development and another one for production. These scripts are supposed to inform in which [mode](https://webpack.js.org/configuration/mode/) Webpack will run. We will have something like this:

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
  },
```

- [ ] We also need to inform users about these script in the README file. This information can be placed in the Usage section. This way users will know how they can use your application.

- [ ] The test script can be safely deleted since we have no tests implemented.

**OPTIONAL**

- [ ]  You can create an even better script for development stages, using the `
webpack-dev-server` and create a script like this:
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```
Running the `start` script will automatically open the index file in the browser, without the need to open it manually